desktop/src/input.rs

```diff
@@ -0,0 +1,142 @@
+use ruffle_core::backend::input::InputBackend;
+use ruffle_core::events::KeyCode;
+use std::collections::HashSet;
+use winit::event::{ElementState, KeyboardInput, VirtualKeyCode};
+
+pub struct GlutinInputBackend {
```

Comment:
Can we rename this `WinitInputBackend`? Glium and Glutin are going to go away in the not-too-far future when I switch to a gfx-rs, and this will just use winit.